global.json

```diff
@@ -1,6 +1,6 @@
 {
-  "projects": [ "src" ],
-  "sdk": {
-    "version": "1.0.0-preview2-1-003177"
-  }
+    "projects": [ "src" ],
+    "sdk": {
+            "version": "1.0.0-rc4-004771"
+    }
```

Comment:
yes you want to pin to specific tooling version as there is no guarantee it will not break with future releases. 

plus if you are missing the right sdk version, global.json ensures we don't attempt to build with an older one and explode